docs/usage.md

```diff
@@ -308,6 +310,7 @@ For starting from the imputation steps, the required flags are:
 | `QUILT`    | ✅               | ✅ ²      | ✅                      | ❌        | ✅         | ✅ ⁴        |
 | `STITCH`   | ✅               | ✅ ²      | ✅                      | ❌        | ❌         | ✅ ³        |
 | `BEAGLE5`  | ✅               | ✅ ¹      | ✅                      | ✅        | ❌         | ❌          |
+| `MINIMAC4` | ✅               | ✅ ¹      | ✅                      | ✅        | ❌         | ❌          |
```

Comment:
Here minimac4 does not require a posfile

Reply:
It doesn't need it.
This is now fixed in the following documentation.